src/applications/disability-benefits/all-claims/content/disabilityLabels.js

```diff
@@ -1,12 +1,5 @@
-import environment from '@department-of-veterans-affairs/platform-utilities/environment';
-import disabilityLabelsReduced from './disabilityLabelsReduced';
 import disabilityLabelsRevised from './disabilityLabelsRevised';
-import revisedFormWrapper from './revisedFormWrapper';
 
 export const getDisabilityLabels = () => {
-  if (revisedFormWrapper.isRevisedForm(environment.isStaging())) {
-    return disabilityLabelsRevised;
-  }
-
-  return disabilityLabelsReduced;
```

Comment:
Curious, is the reduced version still in use?

Reply:
Good point, deleted!